src/modules/stores/innovation/innovation.service.ts

```diff
@@ -57,87 +37,61 @@ export class InnovationService {
 
   }
 
-
   getSectionInfo(innovationId: string, sectionId: string, filters: { fields?: ('actions')[] }): Observable<InnovationSectionInfoDTO> {
 
     const qp = {
       ...(filters.fields ? { fields: filters.fields } : {})
     };
 
-    const url = new UrlModel(this.API_INNOVATIONS_URL).addPath('v1/:innovationId/sections/:sectionId')
-      .setPathParams({
-        innovationId,
-        sectionId
-      }).setQueryParams(qp);
+    const url = new UrlModel(this.API_INNOVATIONS_URL).addPath('v1/:innovationId/sections/:sectionId').setPathParams({ innovationId, sectionId }).setQueryParams(qp);
     return this.http.get<InnovationSectionInfoDTO>(url.buildUrl()).pipe(take(1), map(response => response));
-  }
 
+  }
 
   updateSectionInfo(innovationId: string, sectionKey: string, data: MappedObjectType): Observable<MappedObjectType> {
 
-    const body = { ...data };
     const url = new UrlModel(this.API_INNOVATIONS_URL).addPath('v1/:innovationId/sections/:sectionKey').setPathParams({ userId: this.authenticationStore.getUserId(), innovationId, sectionKey });
-    return this.http.put<MappedObjectType>(url.buildUrl(), body).pipe(
-      take(1),
-      map(response => response)
-    );
+    return this.http.put<MappedObjectType>(url.buildUrl(), data).pipe(take(1), map(response => response));
 
   }
 
   submitSections(innovationId: string, sectionKey: string): Observable<any> {
     const url = new UrlModel(this.API_INNOVATIONS_URL).addPath('v1/:innovationId/sections/:sectionKey/submit').setPathParams({ userId: this.authenticationStore.getUserId(), innovationId, sectionKey });
-    return this.http.patch<any>(url.buildUrl(), { sectionKey }).pipe(
-      take(1),
-      map(response => response)
-    );
+    return this.http.patch<any>(url.buildUrl(), { sectionKey }).pipe(take(1), map(response => response));
   }
 
 
   getSectionEvidenceInfo(innovationId: string, evidenceOffset: string): Observable<GetInnovationEvidenceDTO> {
```

Comment:
Change `evidenceOffset` param to `evidenceId` to maintain consistency

Reply:
Solved. Thanks.